aes256.rs

```diff
@@ -0,0 +1,338 @@
+// Import hacspec and all needed definitions.
+use hacspec::prelude::*;
+// TODO: move to hacspec_imports if we want to use it!
+use contracts::*;
+// I need 32 bytes for a AES256 key CHANGED FROM 16 to 32
+const BLOCKSIZE: usize = 32;
```

Comment:
The block size doesn't change.

Reply:
But the key length has to be 32bytes for aes256, right?